Source/Instrument Addon.lvproj

```diff
@@ -594,11 +607,14 @@
 			<Item Name="NationalInstruments.VeriStand.XMLReader" Type="Document" URL="NationalInstruments.VeriStand.XMLReader">
 				<Property Name="NI.PreserveRelativePath" Type="Bool">true</Property>
 			</Item>
+			<Item Name="nisyscfg.dll" Type="Document" URL="nisyscfg.dll">
+				<Property Name="NI.PreserveRelativePath" Type="Bool">true</Property>
+			</Item>
 			<Item Name="systemLogging.dll" Type="Document" URL="systemLogging.dll">
 				<Property Name="NI.PreserveRelativePath" Type="Bool">true</Property>
 			</Item>
-			<Item Name="VeriStandTestCase.lvclass" Type="LVClass" URL="../../../niveristand-custom-device-testing-tools/VeriStandTestCase/VeriStandTestCase.lvclass"/>
-			<Item Name="VeriStandTestUtilities.lvlib" Type="Library" URL="../../../niveristand-custom-device-testing-tools/VeriStandTestUtilities/VeriStandTestUtilities.lvlib"/>
+			<Item Name="VeriStandTestCase.lvclass" Type="LVClass" URL="../../../niveristand-custom-device-testing-tools-master/VeriStandTestCase/VeriStandTestCase.lvclass"/>

```

Comment:
- Are the changes to the .lvproj needed? The change in line 616 and 617 for sure will break all the tests that use VeriStand Test Utilities.

Reply:
No, these changes are not needed. Also, no, it would not break anything - the only thing you would get is a warning at load time:

![image](https://user-images.githubusercontent.com/35329597/91538263-30a8b280-e8dd-11ea-96ab-6860277933e5.png)

The structure that I have on the dev machine is similar to what we have on the ATS ... still, sure, I guess no big deal reverting.

Reply:
The VeriStand Test Utilities folder names differ on ATS and your setup:
- niveristand-custom-device-testing-tools -> ATS
- niveristand-custom-device-testing-tools**-master** -> your setup

So if we submit this change and run the Tests on the ATS, every test that uses the VeriStand Test Utilities will fail. Additional, every Test Class that uses inheritance from the VeriStand Test Utilities will not load (check below snip):

![Error Loading Test Class if path is different](https://user-images.githubusercontent.com/62251693/91542442-8ce3f480-e926-11ea-92a5-77e66d7707d2.png)